io-engine/src/bdev/nvmx/controller.rs

```diff
@@ -666,31 +665,20 @@ impl<'a> NvmeController<'a> {
             return;
         }
 
-        let rc =
-            unsafe { spdk_nvme_ctrlr_reset(reset_ctx.spdk_handle.as_ptr()) };
-        if rc != 0 {
-            error!(
-                "{} failed to reset controller, rc = {}",
-                reset_ctx.name, rc
-            );
-
-            NvmeController::_complete_reset(reset_ctx, rc);
-        } else {
-            debug!(
-                "{} controller successfully reset, reinitializing I/O channels",
-                reset_ctx.name
-            );
+        debug!(
```

Comment:
So we don't need to issue `spdk_nvme_ctrlr_reset` now?

Reply:
spdk_nvme_ctrlr_reset API includes both the logic for tearing down and re-establish connection. But, by this place, we had already torn down connections for all the channels and we have our own code for reestablishing the connection.